core/server/data/schema/schema.js

```diff
@@ -390,6 +390,15 @@ module.exports = {
         updated_at: {type: 'dateTime', nullable: true},
         updated_by: {type: 'string', maxlength: 24, nullable: true}
     },
+    members_stripe_info: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        member_id: {type: 'string', maxlength: 24, nullable: false, unique: false},
```

Comment:
Out of curiosity have a small Q about the use-cases :)  I get all of them except: `they signup
twice` Why is this possible?

Reply:
Some flows allow to go to payment first and then create a member, at the time of payment we have no way of stopping them if they've already made a payment before! So that would result in a single member with two stripe payments set up

Reply:
do we do anything about such "double" payments?

Reply:
nope!